docs/TutorEnvironmentManager.rst

```diff
@@ -0,0 +1,108 @@
+Tutor Environment Manager
+#########################
+
+Tutor environment manager allows you to create one project and set one version for it.
+
+You will be able to continue using the `tutor version manager`_ commands in your project:
+
+- tvm list
+- tvm plugins list
+
+
+Creating a project
+------------------
+
+To create a new project you should call ``tvm project init``, this configures a new tvm project in the current path and set your-project as ``TUTOR_ROOT`` and ``TUTOR_PLUGINS_ROOT``.  
+
+You can create different projects with the same tutor version, you have 2 options:
```

Comment:
maybe we need to say that if you don't write a name, TVM will create a project in the current fonder and with the name of the current folder, however, if you define a name TVM will create a new folder with that name and in that folder will be created the project.